src/plugins/pluginManager/proposal.ts

```diff
@@ -88,6 +90,9 @@ export class PluginManagerProposal extends Proposal<IPluginManagerProposalState>
       pluginRegistered: item.schemeFactory.schemeRegistered,
       pluginRemoved: item.schemeFactory.schemeRemoved,
       pluginToRegisterData: item.schemeFactory.schemeToRegisterData,
+      pluginToRegisterDecodedData: item.schemeFactory.schemeToRegisterName !== NULL_ADDRESS ?
```

Comment:
does this condition is require after updating `decodeDataByPluginName` ?

Reply:
In `decodeDataByPluginName` we check for `null` and `''`, it's not `NULL_ADDRESS `which is `"0x0000000000000000000000000000000000000000"`. Actually I can move the `NULL_ADDRESS` checking into `decodeDataByPluginName`.

Reply:
In a second thought `NULL_ADDRESS` checking is not necessary because we check name, not address. I used it before when I passed to the function `schemeToRemove`

Reply:
schemeToRegisterName is a string . why competing that with NULL_ADDRESS ?